src/iframe.js

```diff
@@ -1,17 +1,28 @@
 import { render } from '@wordpress/element';
 import { useBlockProps } from '@wordpress/block-editor';
 
-export default function Wikiloc( props ) {
+export default function Wikiloc( attributes ) {
 
-    const result = props.mapUrl.match(/\d+$/);
+    const { 
+        mapUrl,
+        mapWidth,
+        mapHeight,
+        mapType,
+        showTitle,
+        showMeasuresElevation,
+        showLocation,
+        showImages
+    } = attributes.attributes;
```

Comment:
I think it's a good idea to set defaults for these configs, just in case.

```suggestion
    const { 
        mapUrl = '',
        mapWidth = 500,
        mapHeight = 400,
        mapType = 'H',
        showTitle = false,
        showMeasuresElevation = false,
        showLocation = false,
        showImages = false,
    } = attributes.attributes;
```

Reply:
Thanks for suggesting this! I've tested this and the default values are assigned on the [`src/block.json`](https://github.com/jartes/wikiblock/blob/main/src/block.json#L13 ) file. So, I think that in this case, it's better to keep the default values only on the block attributes file.

Reply:
Makes sense!